include/translatable-object.php

```diff
@@ -380,13 +402,38 @@ public function get_objects_with_no_lang( $limit, array $args = array() ) {
 		}
 
 		$key          = md5( $sql );
-		$last_changed = wp_cache_get_last_changed( $this->cache_type );
+		$last_changed = $this->object_cache->get_last_changed();
 		$cache_key    = "{$this->cache_type}_no_lang:{$key}:{$last_changed}";
-		$object_ids   = wp_cache_get( $cache_key, $this->cache_type );
+		$cache_args   = array(
+			'key'   => $cache_key,
+			'group' => $this->cache_type,
+		);
+		$object_ids   = $this->object_cache->get(
+			$this->object_cache->filter_get_args(
+				$cache_args,
+				array(),
+				$this->type,
+				$this->tax_language,
+				$this->tax_to_cache
+			)
+		);
 
 		if ( ! is_array( $object_ids ) ) {
 			$object_ids = $GLOBALS['wpdb']->get_col( $sql ); // PHPCS:ignore WordPress.DB.PreparedSQL.NotPrepared
-			wp_cache_set( $cache_key, $object_ids, $this->cache_type );
+			$cache_args = array(
+				'key'   => $cache_key,
+				'data'  => $object_ids,
+				'group' => $this->cache_type,
+			);
```

Comment:
Since we already have the `key` and the `group` above, can we just add `data` to `$cache_args` ?

Reply:
Done in 9f50fa1.